pkg/alertmanager/alertmanager_metrics.go

```diff
@@ -168,6 +175,34 @@ func newAlertmanagerMetrics() *alertmanagerMetrics {
 			"cortex_alertmanager_state_replication_failed_total",
 			"Number of times we have failed to replicate a state to other alertmanagers",
 			[]string{"user"}, nil),
+		fetchReplicaStateTotal: prometheus.NewDesc(
```

Comment:
is it really useful to have the split of these metrics per `user`? I've the feeling we're adding cardinality without much value.